src/renderer.js

```diff
@@ -113,106 +133,21 @@ Renderer.prototype.getUniforms = function () {
       100
   );
 
-  this.uniforms.u_near = 1;
-  this.uniforms.u_far = 100;
-  this.uniforms.u_view = m4.identity([]);
-  this.uniforms.u_camera = m4.invert([], this.uniforms.u_view);
-  this.uniforms.u_world = camera.computedMatrix;
-  this.uniforms.u_worldView = m4.multiply([], this.uniforms.u_view, this.uniforms.u_world);
-  this.uniforms.u_worldViewProjection = m4.multiply([], projection, this.uniforms.u_worldView);
-
-  return this.uniforms;
-};
+  uniforms.u_near = 1;
+  uniforms.u_far = 100;
+  uniforms.u_view = m4.identity([]);
+  uniforms.u_camera = m4.invert([], uniforms.u_view);
+  uniforms.u_world = camera.computedMatrix;
+  uniforms.u_worldView = m4.multiply([], uniforms.u_view, uniforms.u_world);
+  uniforms.u_worldViewProjection = m4.multiply([], projection, uniforms.u_worldView);
```

Comment:
(All I mean is that relying on the indexing allocation or whatever it's called has some mildly off flavor. =P)